.pre-commit-config.yaml

```diff
@@ -19,24 +19,24 @@ repos:
       - id: trailing-whitespace
 
   - repo: https://github.com/pre-commit/mirrors-prettier
```

Comment:
This repo has the notice on it: This repository has been archived by the owner on Apr 11, 2024. It is now read-only. 

Are we still supposed to use it (the mirror), or is there a way to use prettier directly?

For now, I would ignore this version update.